tests/core/test_configuration.py

```diff
@@ -641,9 +641,9 @@ def test_auth_backends_adds_session(self):
 
     def test_command_from_env(self):
         test_cmdenv_config = """[testcmdenv]
-itsacommand = NOT OK
-notacommand = OK
-"""
+            itsacommand = NOT OK
```

Comment:
This changes the actual string though. It may still "work", but it's also not how ini is typically... Probably better to use dedent.

Reply:
Thanks for the quick review! I thought in any indentation, it would be a string in the Python context. In this context, even spaces between values do not fit into the ini file or any source file, updated that too since dedent won't update that part.

Reply:
How you added dedent doesn't actually change the string :)

dedent removes "common" leading whitepspace. But your first line has none, so nothing gets removed.

Reply:
Yes, my bad! I think my brain is exhausted today :) Calling the day off. I applied the suggestions and escaped the first empty line with `\`.

Reply:
haha no worries!